graphics/src/gradient.rs

```diff
@@ -85,4 +94,28 @@ impl Linear {
 
         self
     }
+
+    /// Packs the [`Gradient`] for use in shader code.
+    pub fn pack(&self) -> [f32; 44] {
```

Comment:
Let's make an opaque `Packed` type in this module and use that instead of `[f32; 44]` everywhere. We'll keep the internal details hidden that way.

Reply:
Updated in 413526ad09d006853eb9659efabee168f4a0e0a4